### PR TITLE
[CloudMinds] 10355-General Info

### PR DESCRIPTION
**Pull request by Kenny-SZ**
This LWM2M Object contains the referrence to all general info objects.

### CODE REVIEW

File: 10355.xml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+<LWM2M xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://openmobilealliance.org/tech/profiles/LWM2M.xsd">
+  <Object ObjectType="MODefinition">
+    <Name>General Info</Name>
```
Comment (akeranen) [inline 10355.xml:35]: This is too generic name for such specific object. Better e.g., "Robot Info"?

File: 10355.xml [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    <Description1><![CDATA[This LWM2M Object contains the referrence to all general info objects.]]></Description1>
+    <ObjectID>10355</ObjectID>
+    <ObjectURN>urn:oma:lwm2m:x:10355</ObjectURN>
+    <LWM2MVersion>1.0</LWM2MVersion>
+    <ObjectVersion>1.0</ObjectVersion>
+    <MultipleInstances>Single</MultipleInstances>
+    <Mandatory>Optional</Mandatory>
+    <Resources>
+	  <Item ID="100">
+		<Name>Robot General Info</Name>
```
Comment (akeranen) [inline 10355.xml:45]: Name doesn't seem to match the purpose of the resource

File: 10355.xml [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+	  <Item ID="101">
+		<Name>RCU General Info</Name>
+		<Operations>R</Operations>
+		<MultipleInstances>Single</MultipleInstances>
+		<Mandatory>Mandatory</Mandatory>
+		<Type>Objlnk</Type>
+		<RangeEnumeration/>
+		<Units/>
+		<Description>
+		  <![CDATA[Contains the reference to the RCU of the Robot.]]>
```
Comment (akeranen) [inline 10355.xml]: Please expand the acronyms here and in all other places in the model descriptions; for short acronyms consider expanding also in names.

File: 10355.xml [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+<LWM2M xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://openmobilealliance.org/tech/profiles/LWM2M.xsd">
+  <Object ObjectType="MODefinition">
+    <Name>General Info</Name>
+    <Description1><![CDATA[This LWM2M Object contains the referrence to all general info objects.]]></Description1>
```
Comment (akeranen) [inline 10355.xml]: ..of robots

**Review by akeranen — CHANGES_REQUESTED** [review]

File: 10355.xml (hunk 4 above)
Comment (Kenny-SZ) [reply 10355.xml]: Thanks for your comments, updated.

**Pull request merged**